release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3362,6 +3362,35 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.41
+[id="ocp-4-16-41_{context}"]
+=== RHBA-2025:8116 - {product-title} {product-version}.41 bug fix and security update
+
+Issued: 28 May 2025
+
+{product-title} release {product-version}.41 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:8116[RHBA-2025:8116] advisory. 
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.41 --pullspecs
+----
+
+[id="ocp-4-16-41-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when you viewed the list of installed Operators, an Operator appeared twice in the list because the currently selected project matched the default namespace of the Operator while the copied cluster service versions (CSVs) were disabled in the Operator Lifecycle Manager (OLM). With this release, the Operator appears only once. (link:https://issues.redhat.com/browse/OCPBUGS-55644[OCPBUGS-55644]).
+
+* Previously, the Assisted Installer failed to detect World Wide Name (WWN) details during Fibre Channel multipath volumes hardware discovery. As a result, a Fibre Channel multipath disk could not be matched with a WWN root device. This meant that when you specified a `wwn` root device hint, the hint excluded all Fibre Channel multipath disks. With this release, the Assisted Installer now detects WWN details during Fibre Channel multipath disk discovery. If multiple Fibre Channel multipath disks exist, you can now use the `wwn` root device hint to choose a primary disk for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-55443[OCPBUGS-55443])
```

Comment:
The first time you say "WWN root device", but the second time and third time you say `wwn` root device (backticks).